metron-deployment/vagrant/Kerberos-setup.md

```diff
@@ -107,23 +107,23 @@ ${HDP_HOME}/kafka-broker/bin/kafka-topics.sh --zookeeper ${ZOOKEEPER}:2181 --cre
 12. Setup Kafka ACLs for the topics
   ```
 export KERB_USER=metron;
-for topic in bro enrichments indexing snort; do
+for topic in bro enrichments indexing snort yaf; do
```

Comment:
Did you want to start `yaf` because the instructions use the YAF topology for validation later on?

Another option, is to not start YAF here (as we know resources are constrained in Full/Quick Dev) and simply change the instructions below to validate against Snort or Bro, rather than YAF.